src/main/java/io/javalin/Javalin.java

```diff
@@ -375,6 +371,15 @@ public Javalin requestLogger(@NotNull RequestLogger requestLogger) {
         return this;
     }
 
+    public Javalin enableMetrics() {
+        ensureActionIsPerformedBeforeServerStart("Enable application metrics");
+        Util.INSTANCE.ensureDependencyPresent(OptionalDependency.MICROMETER);
+        final StatisticsHandler statisticsHandler = new StatisticsHandler();
+        this.jettyServer.insertHandler(statisticsHandler);
+        JavalinMetrics.createInstanceIfNeeded(statisticsHandler, this.jettyServer.getThreadPool());
```

Comment:
> Is this object ever used again?

@tipsy you are right, the object was never used again. Also it was possible to create multiple instances by calling `enableMetrics()` more than once - this is resolved now.